packages/hub/services/relay.ts

```diff
@@ -82,6 +82,33 @@ export default class RelayService {
     let body = await response.json();
     return body.txHash;
   }
+
+  async registerProfile(userAddress: string, did: string): Promise<string> {
+    let relayUrl = getConstantByNetwork('relayServiceURL', web3config.layer2Network);
+    if (!provisionerSecret) {
+      throw new Error(`Could not register profile because relay.provisionerSecret config is not set.`);
+    }
+    let response = await fetch(`${relayUrl}/v1/profiles/register?eoa=${userAddress}&did=${did}`, {
```

Comment:
Current URL that I have in my local relayer feature branch is `v1/merchant/register`. Params are `owner` and `info_did`.

Reply:
Updated now, thanks.